.github/workflows/update-labels-on-change.yml

```
@@ -0,0 +1,16 @@
+  push:
+    branches: [ main ]
+    paths:
+      - '.github/labels.json'
+
+permissions:
+  issues: write
+
+jobs:
+  update-labels:
+    uses: ./.github/workflows/labels.yml
+    with: {}
```

actionlint (1.7.7)

"with" section should not be empty. please remove this section if it's unnecessary

(syntax-check)